bedrock/stories/templates/stories/articles/build-together.html

```diff
@@ -45,23 +45,28 @@
-  <p>For consumers, the open source mindset is no small matter. Not everyone has the skills or desire to open the hood and see how a browser or app works, but knowing that others are doing so should be reassuring. “Anyone can look at the code and know exactly what it does, which is an extraordinary level of corporate transparency,” said Nazer. “If we make a promise about the privacy and security features of Firefox, we’re not asking for blind trust. You can actually go to our repositories and look at the code. That makes us accountable.”</p>
-  <p>Open source culture manifests at Mozilla in other important ways. Many employees join the company in the first place because of its mission, one that guides their daily choices. Said Product Manager Ray Fambro: “I really do believe that something as amazing as the internet should be managed and developed with a greater good and with everyone.”</p>
+  <div class="stories-max-width-wrapper">
+    <h3 class="c-stories-article-section-title">Benefits for All</h3>
+    <p>Firefox was built on the open source model, as were other early internet projects including the Linux kernel and the Apache web server. Though available to all, open source software comes with a license that sets some guidelines about how it can be used and shared.</p>
+    <p>The key aspect of Mozilla’s license is that it is a “copyleft” license, said Senior IP and Product Counsel Daniel Nazer. In contrast to a proprietary approach, where a company fully reserves its copyright, a copyleft license allows anyone to use, distribute, or modify the source code as long as they comply with the terms of the license. That freedom is available to individuals and businesses alike. </p>
+    <p>Legend has it that the idea of <a href="https://www.altlegal.com/blog/to-the-left-to-the-left-an-introduction-to-copyleft-and-open-source-software/" rel="external">copyleft began</a> in the 1970s when a developer added it to the distribution notice of BASIC programming language as a dig at Bill Gates, who’d complained about people pirating Microsoft’s Altair BASIC program. About a decade later, it  took on legal significance when it was used in the distribution license for the <a href="https://www.gnu.org/" rel="external">GNU Project</a>. That license was followed by several others including the <a href="{{ url('mozorg.mpl.index') }}">Mozilla Public License</a> (MPL), which was released in 1998, with <a href="{{ url('mozorg.mpl.2.0.index') }}" rel="external">Version 2.0</a> released in 2010.</p>
+    <p>The MPL includes the crucial requirement that modified versions of MPL-licensed code has to remain open source as well. “The license is designed to create a community around the code, and an infrastructure everyone can use to build and share products,” said Nazer.</p>
```

Comment:
**grammar nit**: I think this is referring to a plural noun so should be "modified versions [...] have" instead of "has"
```suggestion
    <p>The MPL includes the crucial requirement that modified versions of MPL-licensed code have to remain open source as well. “The license is designed to create a community around the code, and an infrastructure everyone can use to build and share products,” said Nazer.</p>
```